jdaviz/configs/default/plugins/line_lists/line_lists.py

```diff
@@ -304,6 +311,11 @@ def _update_line_list_obs(self, *args):
                 else:
                     line_list['lines'][i]['obs'] = self._rest_to_obs(float(line['rest']))
 
+                # Keep a copy of the observed wavelength in the base unit for this plugin
+                line_list['lines'][i]['obs_base'] = (line_list['lines'][i]['obs'] * u.Unit(
+                                                        line_list['lines'][i]['unit'])
+                                                      ).to(BASE_WAVE_UNIT).value
```

Comment:
This is a better API. This comment applies to all relevant code in this PR.
```suggestion
                                                      ).to_value(BASE_WAVE_UNIT)
```